### ISSUE TITLE
Additional tensorial properties: pair with QCEl#241

### ISSUE THREAD

**Issue by bgpeyton** [MolSSI/QCSchema#78]
This is the sister-issue to [QCEl#241](https://github.com/MolSSI/QCElemental/issues/241). If the properties are to be added to the schema, it looks like we should solicit opinions from this repo as well (if I read #68 correctly). 

We're looking to add some tensorial properties, namely polarizabilities and other (linear) response tensors such as the Rosenfeld mixed magnetic- and electric-dipole tensor. This is somewhat related to #70 in that these tensors are often redundant, but also unique in that the details of the perturbing field (order, operator(s), frequency) should also be considered. The [openrsp](http://www.openrsp.org/en/latest/index.html) model is promising, but perhaps more complicated than we need (for now). Any feedback is appreciated.

**Comment by ghutchis** [comment]
Yes, polarizabilities and quadrupole moments would be useful for a range of purposes.

**Comment by ringholm** [comment]
@robertodr told me about this issue. I am with OpenRSP and I am happy to answer any questions you may have about how we approached this.
